packages-ts/starknet-gauntlet-ocr2/src/commands/ocr2/setPayees.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  CONTRACT_TYPES,
+  ExecuteCommandConfig,
+  ExecutionContext,
+  makeExecuteCommand,
+  getRDD,
+} from '@chainlink/starknet-gauntlet'
+import { ocr2ContractLoader } from '../../lib/contracts'
+
+type PayeeConfig = {
+  transmitter: string
+  payee: string
+}
+
+type UserInput = PayeeConfig[]
+
+type ContractInput = [payees: PayeeConfig[]]
+
+const makeUserInput = async (flags, args): Promise<UserInput> => {
+  if (flags.input) return flags.input as UserInput
+  if ((args.length - 1) % 2 != 0) throw new Error('Invalid number of arguments for payee config')
+  let userInput: UserInput = []
+  for (let i = 1; i < args.length; i += 2) {
+    // note: skip first arg which is contract address
+    userInput.push({
+      transmitter: args[i],
+      payee: args[i + 1],
+    })
+  }
+  return userInput
+}
+
+const makeContractInput = async (
+  input: UserInput,
+  ctx: ExecutionContext,
+): Promise<ContractInput> => {
+  if (ctx.rdd) {
+    const aggregator = ctx.rdd[CONTRACT_TYPES.AGGREGATOR][ctx.contractAddress]
+    input = aggregator.payees
+  }
+
+  return input.map((payeeConfig) => [
+    {
+      transmitter: payeeConfig.transmitter,
+      payee: payeeConfig.payee,
+    },
+  ]) as ContractInput
+}
+
+const commandConfig: ExecuteCommandConfig<UserInput, ContractInput> = {
+  contractId: 'ocr2',
```

Comment:
Check other commands, contractId and category aren't normally hardcoded.

Reply:
I moved category into a `constants.ts`, though other commands seem to import from gauntlet-core, in which [contractID](https://github.com/smartcontractkit/gauntlet-core/blob/293122f4f75016e1e77c3107da3cb5f756125857/packages/contracts-ocr2/src/commands/setBilling.ts#L33) does seem to be hardcoded 🤔

Reply:
Other command sets within the same package do operate with contract_list and categories - [example](https://github.com/smartcontractkit/chainlink-starknet/blob/62ee8f90498188ca9a4d0f2ae3060244bcbf0111/packages-ts/starknet-gauntlet-ocr2/src/commands/proxy/proposeAggregator.ts#L45)

Reply:
ah thanks, missed that one 👍 just noting that there are some inconsistencies between commands that import from gauntlet-core (contractID = 'ocr2') and commands that import locally from lib/contracts (contractID = 'Aggregator') but not sure if there are any implications and will leave it for now